tasks/install-with-package.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install Certbot.
+  package: name=certbot state=present
```

Comment:
I'm guessing it would be better to update the failing tests to use from-source at this point? Or is the plan to continue updating the project to use the alternative repositories and/or alternative binary names for the failing distros?